docs/en/differenze_oidc_fed.rst

```diff
@@ -11,19 +11,13 @@ This section lists the differences between the official standard and the SPID /
 Client Registration
 +++++++++++++++++++
 
-SPID supports only **automatic_client_registration**. The **implicit** flow is not supported.
-
-
-Listing endpoint
-++++++++++++++++
-
-In the SPID / CIE Federations, for this endpoint, the claim **entity_type** is added to all the other `OIDC-FED`_ standard claims, in order to obtain a filter on the type of subordinate Entities. This need apecifically allows filtering Entities of type **federation_entity**, **openid_relying_party**, **openid_provider** and **oauth_resource**.
+SPID and CIE support only **automatic_client_registration**. The **explicit client registration** flow is not supported.
```

Comment:
**explicit client registration** --> **explicit_client_registration**

Reply:
Non sarebbe meglio usare un linguaggio più nrmativo qui? .... MUST support ... MUST NOT support ....

Reply:
se i soggetti sono SPID e CIE non hanno MUST, i MUST ce li hanno i partecipanti/implementatori

Reply:
a mio avviso il MUST vale sia per SPID CIE che per i partecipanti/implementatori. Ripensandoci, però, essendo in una sezione di differenze, forse potremmo anche evitare di usare linguaggio normativo qui.

Reply:
yes, le sezioni di "considerazioni" non hanno linguaggio normativo